docs/execution-providers/Cloud-ExecutionProvider.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cloud
+description: Instructions to infer an ONNX model remotely with a cloud endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Cloud-ExecutionProvider
+---
+
+# Cloud Execution Provider
+
+The Cloud Execution Provider enables ONNX Runtime to invoke a cloud endpoint for inferenece, the endpoint must be deployed beforehand.
```

Comment:
```suggestion
The Cloud Execution Provider enables ONNX Runtime to inference a model in the cloud. The model needs to be deployed to the cloud before using ORT.
```

Reply:
Also, please explicitly mention the use case (small/big model) this preview is targeting.

Reply:
Added small-big model case.